jelly-tags/xml/src/test/java/org/apache/commons/jelly/tags/xml/TestXMLTags.java

```diff
@@ -43,6 +43,11 @@
 import org.dom4j.Node;
 import org.dom4j.io.SAXContentHandler;
 import org.dom4j.io.XMLWriter;
+import org.hamcrest.CoreMatchers;
+import org.hamcrest.MatcherAssert;
+
+import static org.hamcrest.CoreMatchers.startsWith;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
The existing annotation failed at first without a useful message, so I switched to Hemcrest as it reports a difference.